deploy/docker/Dockerfile

```diff
@@ -29,9 +29,19 @@ COPY --from=builder /emqx-rel/emqx /opt/emqx
 
 RUN ln -s /opt/emqx/bin/* /usr/local/bin/
 
-RUN apt-get update; \
-    apt-get install -y --no-install-recommends ca-certificates procps; \
-    rm -rf /var/lib/apt/lists/*
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates procps
+
+RUN apt-get update \
+    && apt-get install -y gnupg2 curl apt-utils \
+    && curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+    && apt-get update \
+    && ACCEPT_EULA=Y apt-get install -y msodbcsql17 unixodbc-dev \
```

Comment:
@JimMoen are you sure it's a good idea to auto-accept EULA and include this in our standard docker image?
May be we could tell people instead how to extend our docker image to add msodbcsql17 driver if they need it?